### PR TITLE
ibu cnf: add prep abort test case

### PR DESCRIPTION
**Pull request by mpmaruthu**
adding ibu prep abort test case

### CODE REVIEW

**Comment by mpmaruthu** [comment]
hello, @mcornea, I added the upgrade prep abort test case and addressed initial review comments received for the same case. Please take a look when you get a chance. Thank you!

Here is the reference of old PR and comments,
`https://github.com/openshift-kni/eco-gotests-old/pull/748`

**Comment by mcornea** [comment]
lgtm

**Comment by mpmaruthu** [comment]
hello @klaskosk, I added the upgrade prep abort test case and addressed your review comments received for the same case. Please take a look when you get a chance. Thank you!

Here is the reference of old PR and comments,
https://github.com/openshift-kni/eco-gotests-old/pull/748

**Review by klaskosk — CHANGES_REQUESTED** [review]
Just two minor comments

File: tests/lca/imagebasedupgrade/cnf/upgrade-talm/tests/prep-abort-test.go [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+			By("Fetching target sno cluster name", func() {
+				err := cnfclusterinfo.PreUpgradeClusterInfo.SaveClusterInfo()
+				Expect(err).ToNot(HaveOccurred(), "Failed to extract target sno cluster name")
+
+				tsparams.TargetSnoClusterName = cnfclusterinfo.PreUpgradeClusterInfo.Name
+
+			})
+		})
+
+		AfterAll(func() {
```
[the remark this answers is not in the document]
Comment (mpmaruthu) [reply tests/lca/imagebasedupgrade/cnf/upgrade-talm/tests/prep-abort-test.go:32]: Changes updated. Thanks!

**Review by klaskosk — APPROVED** [review]
/lgtm

**Review by mcornea — APPROVED** [review]

**Pull request merged**